workflow/cases/hofx3d.yaml

```diff
@@ -0,0 +1,18 @@
+case:
+  settings:
+    SDATE: 2011-10-01t12:00:00
+    EDATE: 2011-10-02t12:00:00
+    godas_cyc: 1 # selection of godas cycle: 1(default)- 24h; 2 - 12h; 4 - 6h
+    resolution: Ocean025deg
+    FCSTMODEL: MOM6CICE5
+
+  da_settings:
+    NINNER: 100
```

Comment:
The NINNER is not required for the HOFX, just deleted from this file.

Reply:
Deleted